fem/quadinterpolator_face.cpp

```diff
@@ -495,4 +495,11 @@ void FaceQuadratureInterpolator::Mult(
    }
 }
 
+void FaceQuadratureInterpolator::Values(
+   const Vector &e_vec, Vector &q_val) const
+{
+   Vector q_der, q_det, q_nor;
+   Mult(e_vec, VALUES, q_val, q_der, q_det, q_nor);
```

Comment:
@YohannDudouit , @v-dobrev  using VALUES doesn't work as expected. Is that not the enum constant?

Reply:
Ah hang on, I think I might have a different bug in my application.

Reply:
nvm, it works great.